person.rb

```diff
@@ -1,9 +1,12 @@
 # FrozenStringLiteral: true
-class Person
+require_relative 'nameable'
+
+class Person < Nameable
   attr_reader :id, :parent_permission
   attr_accessor :name, :age
 
   def initialize(name = 'Unknown', age = 18, parent_permission: true)
+    super()
     @id = generate_id
     @name = name
     @age = age
```

Comment:
- [ ] According to the project requirement, this class is supposed to have a method called `correct_name` should should return name attribute. Kindly check the project requirement to confirm this. :point_down: 

### Project requirement 
![correct_name](https://github.com/berylcl/OOP-school-library/assets/26597432/cb9b9fd1-bf62-49d7-be93-34c6d700b7be)